PureBasicIDE/AutoComplete.pb

```diff
@@ -81,7 +81,7 @@ Procedure CreateAutoCompleteWindow()
     ;     AddKeyboardShortcut(#WINDOW_AutoComplete, #PB_Shortcut_Escape, #MENU_AutoComplete_Abort)
     ;     AddKeyboardShortcut(#WINDOW_AutoComplete, #PB_Shortcut_Return, #MENU_AutoComplete_Ok)
     ;     AddKeyboardShortcut(#WINDOW_AutoComplete, #PB_Shortcut_Tab, #MENU_AutoComplete_Ok)
-    
+    AddKeyboardShortcut(#WINDOW_AutoComplete, #PB_Shortcut_Shift|#PB_Shortcut_Tab, #MENU_AutoComplete_Full)
```

Comment:
Can you make the shortcut configurable in the preferences? See `ShortcutManagement.pb` for details.